### PR TITLE
Add test for null values when getting DTV values

### PR DESCRIPTION
**Pull request by Ananya2**
Fix for datatypes-unit fx test case failure while keeping perf fix. Ensure exception is thrown before returning null in getValue() 

Adds test case for #2600

### CODE REVIEW

**Review by machavan — APPROVED** [review]

**Review by divang — DISMISSED** [review]

**Review by machavan — APPROVED** [review]

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java [hunk 1]
```diff
@@ -3078,4 +3080,92 @@ public void terminate() {
         }
     }
 
+    @Nested
+    public class TestPLP {
+        
+        @Test
+        public void testGetAsciiStreamOnXml() {
```
Comment (lilgreenbird) [inline src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java:3087]: this doesn't seem to be a statement test, why is it added here? This is from the internal datatype tests seems more appropriate to be added to datatypes test suite? or lobs where the streams are tested.

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java [hunk 2]
```diff
@@ -3078,4 +3080,92 @@ public void terminate() {
...
+                    }
+                }
+            } catch (SQLException e) {
+                fail("Database setup or execution failed: " + e.getMessage());
+            } finally {
+                try (Connection conn = getConnection();
+                    Statement stmt = conn.createStatement()) {
+                    TestUtils.dropTableIfExists(tableName, stmt);
+                } catch (SQLException ex) {
+                    System.err.println("Failed to clean up test table: " + ex.getMessage());
```
Comment (lilgreenbird) [inline src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java:3124]: this is unnecessary it's handled in dropTableIfExists

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java (hunk 1 above)
Comment (Ananya2) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java]: Refactored tests: Moved datatype-related stream tests to LobsStreamingTest file.

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java (hunk 2 above)
Comment (Ananya2) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java]: Done.

**Review by machavan — APPROVED** [review]

**Review by Jeffery-Wasty — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]

**Review by tkyc — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]
